### DISCUSSION TITLE
Transmission v2.94 with image haugene/transmission-openvpn:latest

### DISCUSSION THREAD

**Discussion by Solamain** [haugene/docker-transmission-openvpn#2424] [Q&A] [answered]
Just curious, thought the latest tag would point to the newest 4.3.x version using Ubuntu 22.04 OVPN 2.5.5 and Transmission 3.0, but it is using OpenVPN 2.4.7, Transmission 2.94 and Ubuntu 20.04.5 LTS is the version running in the container.  If :latest is not the new 4.3.x what tag should be used to always get the latest, as dev is 2 commits behind.  Please forgive me if there is an obvious mistake on my end, as I always find myself looking into these things when I am done working on my own projects so I am mostly burned out by the time I look at these things.

**Comment by pkishino** [comment, marked as answer]
either use a point release and specify ``4.3`` or use ``edge``

**Reply by pkishino** [reply]
latest has been kinda decommissioned as it does not follow proper docker nomenclature
